docs/en/stack/ml/df-analytics/hyperparameters.asciidoc

```diff
@@ -20,9 +20,22 @@ a model, and calculate its performance on validation data.
 
 You can view the hyperparameter values that were ultimately chosen by expanding
 the job details in {kib} or by using the
-{ref}/get-dfanalytics-stats.html[get {dfanalytics-job} stats API]. You can also
-see the specific type of validation loss (such as mean squared error or binomial
-cross entropy) that was used to compare each round of optimization. 
+{ref}/get-trained-models.html[get trained models API]. You can also see the 
+specific type of validation loss (such as mean squared error or binomial cross 
+entropy) that was used to compare each round of optimization using the 
+{ref}/get-dfanalytics-stats.html[get {dfanalytics-job} stats API].
+
+
+[[hyperparameter-importance]]
+== Calculating the importance of hyperparameters
+
+Different hyperparameters may affect the model accuracy to a different degree. 
+To estimate the importance of the optimized hyperparameters, analysis of 
+variance decomposition is used. The resulting `absolute importance` shows how 
+much the variation of a hyperparameter impacts the variation in the validation 
+loss. Additionally, `relative importance` is also computed which gives the 
```

Comment:
> Additionally, `relative importance` is also computed which gives the 

It's not clear to me from reading this where I find these `absolute importance` and `relative importance` values. If you view these details in Kibana or in API results, it likely works to merge this detail into the preceding paragraph.